spec/fabricators/post_fabricator.rb

```diff
@@ -144,3 +144,12 @@
     incoming_email.user = post.user
   end
 end
+
+Fabricator(:post_with_likes, from: :post) do
```

Comment:
I would prefer just to keep this in post_revisor_spec cause its a bit of a magic fabricator and its better to keep the magic all in the file that is doing the testing im